pkgs/top-level/all-packages.nix

```diff
@@ -6281,6 +6281,8 @@ in
 
   cgdb = callPackage ../development/tools/misc/cgdb { };
 
+  cheat = callPackage ../data/documentation/cheat { };
```

Comment:
`data` category is for data (not applications). A better place for `cheat` is `applications/office` or `applications/misc` (IMHO)